src/Apps/Sell/Routes/NewRoute/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react"
+import { useRouter } from "System/Router/useRouter"
+import { SellArtistAutocomplete } from "Apps/Sell/Routes/NewRoute/SellArtistAutocomplete"
+import { useMutation } from "Utils/Hooks/useMutation"
+import { graphql } from "react-relay"
+import { NewRouteMyCollectionCreateArtworkMutation } from "__generated__/NewRouteMyCollectionCreateArtworkMutation.graphql"
+import { NewRouteCreateConsignmentSubmissionMutation } from "__generated__/NewRouteCreateConsignmentSubmissionMutation.graphql"
+
+export const NewRoute: React.FC = () => {
+  const { router } = useRouter()
+  const { submitMutation: submitCreateArtworkMutation } = useMutation<NewRouteMyCollectionCreateArtworkMutation>({
+    mutation: graphql`
+      mutation NewRouteMyCollectionCreateArtworkMutation($artistID: String!) {
+        myCollectionCreateArtwork(input: { artistIds: [$artistID], title: "Unknown" }) {
+          artworkOrError {
+            __typename
+            ... on MyCollectionArtworkMutationSuccess {
+              artwork {
+                internalID
+              }
+            }
+          }
+          clientMutationId
+        }
+      }
+    `,
+  })
+
+  const { submitMutation: submitCreateSubmissionMutation } = useMutation<NewRouteCreateConsignmentSubmissionMutation>({
+    mutation: graphql`
+      mutation NewRouteCreateConsignmentSubmissionMutation($artistID: String!, $myCollectionArtworkID: String!) {
+        createConsignmentSubmission(input: { artistID: $artistID, myCollectionArtworkID: $myCollectionArtworkID }) {
+          clientMutationId
+          consignmentSubmission {
+            internalID
+          }
+        }
+      }
+    `,
+  })
```

Comment:
We should move mutations in to a `Sell/Mutations` folder, following other patterns